### ISSUE TITLE
 Option to disable transcoding, atleast video transcoding.

### ISSUE THREAD

**Issue by Premicheal** [Stremio/stremio-service#71]
I have a mini pc setup to run stremio-service, if any client that cannot support the native format connects, the server goes to 90% cpu load and all streams lag. Is there anyway to turn off transcoding since servers without hw accelerations won't be able to do much anyway, especially arm ones. My apologies if I have missed something.

**Comment by TRtomasz** [comment]
If you disable it you will not be able to play the stream so there is no point in this.

**Comment by TRtomasz** [comment]
@jaruba  will have to answer

**Comment by hyperion4** [comment]
> If you disable it you will not be able to play the stream so there is no point in this.

That's not true. Reasons of transcoding in plex/emby/jellyfin servers.
1. You want to save your upload bandwidth and you "send" from your plex/emby server to your phone a 720p instead of your 4k video file.
2. You have people who  use your server and you don't want them to use the 4k video file you have, you don't want to upload that big files.
3. You want to watch to your browser and the browser doesn't support the video codec, for example hevc.

I don't think 1 or 2 are case scennarios for Stremio.
The 3rd reason.. today all browsers have support for hardware hevc decoding. They have added support the last 2 years. So it is very possible your browser to be capable to play hevc.
I personally have disabled in my emby server video transcoding because my browsers can play everything including hevc.

Unfortunately Stremio service insists to transcode hevc to h264 with ffmpeg even if my browser supports hardware hevc decoding.
Emby does the same and transcodes hevc to h264, but it has the option to disable transcoding and let the browser to try and play hevc itself.

**Comment by aayushrautela** [comment]
Yes, it would make stremio server on low end server useful. Most other devices can play h265.

**Comment by AlvaroRojas** [comment]
This would be very usefull for ARM based servers

**Comment by aayushrautela** [comment]
No. But I think changing server address to stremio web should do the job.

**Comment by Loukious** [comment]
> No. But I think changing server address to stremio web should do the job.

What server address?

**Comment by Loukious** [comment]
> No. But I think changing server address to stremio web should do the job.

Can you explain?

**Comment by webmatrixxxl** [comment]
Make your life easier by adding an option to Disable Transcoding.
